code-challenges.test.js

```diff
@@ -34,13 +65,56 @@ const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 
 // b) Create the function that makes the test pass.
 
+// Pseudo code
+// create a method called remainders that takes in an array as parameter
+// create variable that stores the array filtered by the type using the .filter method.
+// map the filtered array to be able to assign a second variable which I designated as my remain,  then return that var is the remainder of the number divided by 3 
+// return filter val
+//console log to see expected outputs. 
+
+const remainders = (array) => {
+  let filter = array.filter(item => typeof item === 'number' ).map(number => {
+    let remain = number % 3
+    return remain
+  })
+  return filter
+  }
+console.log(remainders(hodgepodge1));
+console.log(remainders(hodgepodge2));
+
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
+
+describe('cuber', () => {
+  it("returns sum of all numbers cubed", () => {
+    expect(cuber(cubeAndSum1)).toEqual(99);
+    expect(cuber(cubeAndSum2)).toEqual(1125);
+  })
+})
+
+
 const cubeAndSum1 = [2, 3, 4]
 // Expected output: 99
 const cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
 // b) Create the function that makes the test pass.
+
+// Pseudo Code
+// create method called cuber that takes the array as a parameter.
+// create variable in which numbers array gets mapped and with the variable of num that returns the values as numbers instead of string by using the parseInt method.
+// return cube number by using exponential operator
+// return the numbers and using the .reduce method we can take all of the numbers within my array and makes it into one element, specifying that I wanted each number to add to the next. 
+// console log to see expected output.
+
+
+const cuber = (numArray) => {
+  let numbers = numArray.map( num => {
+    return parseInt(num) ** 3
```

Comment:
The original array contains numbers already, not strings, so parseInt is not necessary here